crowbar_framework/app/assets/stylesheets/content/panel.scss

```diff
@@ -0,0 +1,56 @@
+.panel-heading {
+  border-bottom: 0;
+}
+
+.panel .panel-heading h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
+  border: 1px;
+  border-color: #ddd;
+  border-left: 7px;
+  border-radius: 5px;
+  border-style: solid;
+  box-shadow: 1px 1px 0px #eee;
+  margin: 5px;
+}
+
+.panel-heading+.list-group .list-group-item:first-child {
+  border-top-width: 1px;
+}
+
+.panel>.list-group:last-child .list-group-item:last-child, .panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{
+  border-bottom: 1px solid #ccc; ;
+}
+
+.status-border-unready {
+  border-left-color: #ddd !important;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Properties should be ordered animation-duration, animation-iteration-count, animation-name, border-left-color